src/js/components/Chart.js

```diff
@@ -217,6 +217,9 @@ export default class Chart extends Component {
 
     let graphWidth = width;
     let graphHeight = height;
+    if (this.props.legend && 'inline' === this.props.legend.position) {
+      graphHeight = graphHeight * 0.9;
```

Comment:
should we use a plus sign instead?

`graphHeight = graphHeight + legendHeight;`

I think there is a way to get the computed legend height.